### PR TITLE
[15.0][MIG] mrp_tag : Migration to 15.0

### PR DESCRIPTION
**Pull request by AnnaPForgeFlow**
@ForgeFlow
Standard migration to 15.0

### CODE REVIEW

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
I think you're mixing migrations.

@LoisRForgeFlow

File: mrp_tag/models/mrp_tag.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from random import randint
+
+from odoo import fields, models
+
+
+class MrpTag(models.Model):
+    _name = "mrp.tag"
+    _description = "mrp Tag"
```
Comment (MiquelRForgeFlow) [inline mrp_tag/models/mrp_tag.py]:
```suggestion
    _description = "MRP Tag"
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**